lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/TestTang.java

```diff
@@ -713,6 +716,27 @@ public void testMultiInheritanceSubclassFirst() throws BindException, InjectionE
     final CheckChildIface o1 = i.getInstance(CheckChildIface.class);
     Assert.assertTrue(o1 instanceof CheckChildImpl);
   }
+
+  /**
+   * Tang supports empty string, '', as a default value.
+   */
+  @Test
+  public void testEmptyStringAsDefaultValue() throws InjectionException {
+    final ConfigurationModule cm = new EmptyStringAsDefaultParamConf()
+        .bindNamedParameter(EmptyStringAsDefaultParam.class, EmptyStringAsDefaultParamConf.OPTIONAL_STRING)
```

Comment:
Oh, I just made this test case according to the our `TimerMock` example. Should I change this according to your comment?